sirepo/package_data/static/js/sirepo-command.js

```diff
@@ -242,6 +251,11 @@ SIREPO.app.directive('commandTable', function(appState, commandService, latticeS
                 return -1;
             }
 
+            $scope.copyCommand = function(cmd) {
+                const copy = $.extend({}, commandService.commandForId(cmd._id), commandService.getNextCommand(cmd._type));
```

Comment:
FYI javascript supports using the spread operator for this so you don't have to rely on jquery:
```javascript
const copy = {...commandService.commandForId(cmd._id), ...commandService.getNextCommand(cmd._type))};
```
You can also skip the temp variable

Reply:
thanks! 77045d74288ceb536b10500920e635863ad3d399